### PR TITLE
Resize images to required display size

### LINKED ISSUE #10907: In-line images not displaying correctly
## What page(s) did you find the problem on?

Various tools pages e.g.[Network Monitor](https://developer.mozilla.org/en-US/docs/Tools/Network_Monitor) where images of icons are used within sentences but by default displayed as block elements and therefore on separate lines.

image

These images need to be updated to include the `style="display: inline"` directive to display them in-line. They might also look better if resized e.g. `height="20" width="20"`

image

### PR DESCRIPTION
**Pull request by AnilSeervi**
#### Summary

- Images by default have `display: block` styling which are displaying images that are expected to be inline -> as block.
#### Motivation

Open-source
#### Related issues

Closes #10907 
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/tools/network_monitor/index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@ <h2 id="Opening_the_Network_Monitor">Opening the Network Monitor</h2>
 <ul>
  <li>Press  <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>E</kbd> ( <kbd>Command</kbd> + <kbd>Option</kbd> + <kbd>E</kbd> on a Mac).</li>
  <li>Select "Network" from the Web Developer menu, (which is a submenu of the Tools menu on OS X and Linux).</li>
- <li>Click the wrench icon (<img alt="" src="wrench-icon.png">), which is in the main toolbar or under the Hamburger menu (<img alt="" src="hamburger.png">), then select "Network".</li>
+ <li>Click the wrench icon (<img alt="" src="wrench-icon.png" style="display: inline;" width=20 height=20>), which is in the main toolbar or under the Hamburger menu (<img alt="" src="hamburger.png" style="display: inline;" width=20 height=20>), then select "Network".</li>
```
Comment (hamishwillee) [inline files/en-us/tools/network_monitor/index.html:24]: Thanks!
1. Shouldn't there be quote marks? i.e.  `width="20"`
2. Not sure what the "rules" are for MDN, but generally I'd just set the width, or even better shrink the image to the actual size required.

Comment (AnilSeervi) [reply files/en-us/tools/network_monitor/index.html:24]: Shrinking the image to required size seems to be a reasonable thing to do here as It'll be helpful when converted to markdown.

**Comment by AnilSeervi** [comment]
I'll shrink the images to required size, eliminating the usage of height and width attribute.
But I also wonder how these images will be placed inline after the content is converted to markdown?

**Comment by hamishwillee** [comment]
> But I also wonder how these images will be placed inline after the content is converted to markdown?

Unlikely. A block element is by definition not displayed inline.

File: files/en-us/tools/network_monitor/index.html (hunk 1 above)
Comment (wbamberg) [reply files/en-us/tools/network_monitor/index.html:24]: FWIW I very much doubt that these pages will be converted into Markdown - it is I believe the intention for them to move off MDN at some point, so we left them out of the Markdown project.

**Comment by AnilSeervi** [comment]
Resized the images and removed the `width` and `height` attributes.

**Review by AnilSeervi — COMMENTED** [review]
Removed inline styling as per mdn/yari#5030

**Comment by wannurhadi** [comment]
// manifest.json

"background": {
  "scripts": ["background-script.js"]
}
// manifest.json

"background": {
  "page": "background-page.html"
}
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <script type="module" src="background-script.js"></script>
  </head>
</html>
Window
The Window interface represents a window containing a DOM document; the document property points to the DOM document loaded in that window.

A window for a given document can be obtained using the document.defaultView property.

A global variable, window, representing the window in which the script is running, is exposed to JavaScript code.

The Window interface is home to a variety of functions, namespaces, objects, and constructors which are not necessarily directly associated with the concept of a user interface window. However, the Window interface is a suitable place to include these items that need to be globally available. Many of these are documented in the JavaScript Reference and the DOM Reference.

In a tabbed browser, each tab is represented by its own Window object; the global window seen by JavaScript code running within a given tab always represents the tab in which the code is running. That said, even in a tabbed browser, some properties and methods still apply to the overall window that contains the tab, such as resizeTo() and innerHeight. Generally, anything that can't reasonably pertain to a tab pertains to the window instead.

**Comment by ddbeck** [comment]
Hi, I'm closing this because [Firefox DevTools User Docs](https://firefox-source-docs.mozilla.org/devtools-user/index.html#firefox-devtools-user-docs) have moved off of MDN (see https://github.com/mdn/content/pull/14140 for details on content moved). If this issue is still applicable to the relocated documentation, please see the Firefox Source Docs site for details on raising bugs and contributing. Thanks for your understanding!

**Pull request closed without merging**